src/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+list(APPEND fortran_src
+  module_fv3gfs_ncio.f90
```

Comment:
Presumably CMake is smart enough to know that there are multiple .f90 files that create this module?

Reply:
yes. The other `.f90` files are "included" in the `module_fv3gfs_ncio.f90` and cannot be compiled separately.